CollapseLauncher/CollapseLauncher.csproj

```diff
@@ -132,18 +131,29 @@
 		</Content>
 
 		<None Include="..\.editorconfig" Link=".editorconfig" />
+
+		<Content Include="..\PRIVACY.md">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</Content>
+
+		<Content Include="..\THIRD_PARTY_NOTICES.md">
```

Comment:
We might need to copy this to separate folder other than root so it doesn't scatter the root folder